time-calculator/time_calculator.py

```diff
@@ -0,0 +1,67 @@
+
+def add_time(startTime, stopTime, dayOfWeek=None):
+
+    # Start Time
+    [startTime, start_am_pm] = startTime.split()
```

Comment:
Here we are mutating `startTime` variable which is passed as a input ..... kindly avoid it in favour of good practices